app/commands_app/commands/get_pods.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+from kubernetes import client, config
+
+def parse_args(sub_parser):
+    subparser = sub_parser.add_parser("get-pods", help="Get all aladdin-demo pods")
+    subparser.set_defaults(func=get_pods)
+
+
+def get_pods(arg):
+    print(get_aladdin_demo_pods())
+
+def get_aladdin_demo_pods():
+    config.load_incluster_config()
+    v1 = client.CoreV1Api()
+    res = v1.list_namespaced_pod(
+        namespace=os.environ["NAMESPACE"],
+        label_selector="project=aladdin-demo")
```

Comment:
why not use `os.environ["PROJECT_NAME"]` since it's defined correctly:
```
$ aladdin connect aladdin-demo-commands
bash-4.3# echo $PROJECT_NAME
aladdin-demo
```